sessions/html-forms/link-headline-and-form/css/styles.css

```
@@ -0,0 +1,22 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	margin: 20px auto;
+	max-width: 300px;
+}
+
+form * {
+	display: block;
+}
```

Might be a bit nicer 😄 
```suggestion
form {
	display: grid;
	gap: 12px;
}
```